src/lib/ical/parse.test.ts

```diff
@@ -15,5 +15,19 @@ test('ICAL parser integration test', () => {
   //   `export const events = ${JSON.stringify(events)};`
   // );
 
-  expect(JSON.stringify(events)).toBe(JSON.stringify(EVENT_DATA));
+  let i = 0;
+  events.forEach((element) => {
+    for (const prop in element) {
+      if (
+        Object.prototype.hasOwnProperty.call(element, prop) &&
+        prop !== 'location' &&
+        prop !== 'calendarLinks'
+      ) {
+        expect(element[prop]).toEqual(EVENT_DATA[i][prop]);
+      }
+    }
+    i += 1;
+  });
+
+  //expect(JSON.stringify(events)).toBe(JSON.stringify(EVENT_DATA));
```

Comment:
FYI, I originally used `JSON.stringify` to hash large lists of JavaScript objects and deterministically verify they matched. However, Vitest seems to give us access to a deep-equality method, `.toEqual()` (instead of `.toBe()`), which may work for this situation.

Reply:
I tried `.toEqual()` and it wasn't seeming to work, will try again tho during this next commit